index.html

```diff
@@ -15,14 +15,22 @@
             <nav class="logo-show">
               <h2>My Logo</h2>
             </nav>
-            <nav class="menu">
-              <nav><img src="photo/navs/Union.png" alt=""></nav>
-            </nav>
+            <div class="menu">
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li class="tul-1">Portfolio</li>
+                <li class="tul-1">About</li>
+                <li class="tul-1">Contact</li>
+              </ul>
+            </div>
```

Comment:
- [ ] As I can see here your menu doesn't cover all the section so kindly stick with the Figma template as much as possible so kindly cover all the section to match the Figma ✅💪🏻


- Your design
![image](https://user-images.githubusercontent.com/58553711/180904633-4f953d2a-090e-41e5-93aa-72df25c6c77a.png)

- Figma template
![image](https://user-images.githubusercontent.com/58553711/180904742-b96cd130-60e5-4238-84a8-2d687587f638.png)